### PR TITLE
Updates for SQLPro products. The downloads now use the latest version…

### PR DESCRIPTION
**Pull request by hankinsoft**
SQLPro for SQLite cask has been re-named to use proper tokens and now uses the latest version.
SQLPro for Postgres cask has been updated to use the latest version.
Added SQLPro for MSSQL cask.
Added SQLPro for MySQL cask.
Added SQLPro Studio cask.
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/sqlpro-for-postgres.rb [hunk 1]
```diff
@@ -1,17 +1,16 @@
 cask 'sqlpro-for-postgres' do
-  version '43'
-  sha256 'ef510f2a7f69d0071872e9942f5f3cb967dadf1840e6ef978b38e8db3b019daf'
+  version :latest
```
Comment (adidalal) [inline Casks/sqlpro-for-postgres.rb]: Use `1.0.46`

File: Casks/sqlpro-for-postgres.rb [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
+  sha256 :no_check
```
Comment (adidalal) [inline Casks/sqlpro-for-postgres.rb]: Needs to be set

File: Casks/sqlpro-for-postgres.rb [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
 
-  # d3fwkemdw8spx3.cloudfront.net was verified as official when first introduced to the cask
-  url "https://d3fwkemdw8spx3.cloudfront.net/postgres/SQLProPostgres#{version}.app.zip"
+  url 'http://www.macpostgresclient.com/download.php'
```
Comment (adidalal) [inline Casks/sqlpro-for-postgres.rb]: Keep this as it was

File: Casks/sqlpro-for-sqlite.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+cask 'sqlpro-for-sqlite' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://www.sqlitepro.com/download.php'
```
Comment (adidalal) [inline Casks/sqlpro-for-sqlite.rb]: Should be https://d3fwkemdw8spx3.cloudfront.net/sqlite/SQLProSQLite.#{version}.app.zip + please add url comment

File: Casks/sqlpro-for-sqlite.rb [hunk 5]
```diff
@@ -0,0 +1,16 @@
+cask 'sqlpro-for-sqlite' do
+  version :latest
```
Comment (adidalal) [inline Casks/sqlpro-for-sqlite.rb]: 1.0.88

File: Casks/sqlpro-studio.rb [hunk 6]
```diff
@@ -0,0 +1,16 @@
+cask 'sqlpro-studio' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.sqlprostudio.com/download.php'
```
Comment (adidalal) [inline Casks/sqlpro-studio.rb]: https://d3fwkemdw8spx3.cloudfront.net/studio/SQLProStudio.#{version}.app.zip

File: Casks/sqlpro-studio.rb [hunk 7]
```diff
@@ -0,0 +1,16 @@
+cask 'sqlpro-studio' do
+  version :latest
```
Comment (adidalal) [inline Casks/sqlpro-studio.rb]: 1.0.27

File: Casks/sqlpro-for-mysql.rb [hunk 8]
```diff
@@ -0,0 +1,16 @@
+cask 'sqlpro-for-mysql' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.mysqlui.com/download.php'
```
Comment (adidalal) [inline Casks/sqlpro-for-mysql.rb]: https://d3fwkemdw8spx3.cloudfront.net/mysql/SQLProMySQL.{version}.app.zip

File: Casks/sqlpro-for-mysql.rb [hunk 9]
```diff
@@ -0,0 +1,16 @@
+cask 'sqlpro-for-mysql' do
+  version :latest
```
Comment (adidalal) [inline Casks/sqlpro-for-mysql.rb]: 1.0.20

File: Casks/sqlpro-for-mssql.rb [hunk 10]
```diff
@@ -0,0 +1,16 @@
+cask 'sqlpro-for-mssql' do
+  version :latest
```
Comment (adidalal) [inline Casks/sqlpro-for-mssql.rb]: 1.0.73

File: Casks/sqlpro-for-mssql.rb [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+  sha256 :no_check
+
+  url 'http://macsqlclient.com/download.php'
```
Comment (adidalal) [inline Casks/sqlpro-for-mssql.rb]: https://d3fwkemdw8spx3.cloudfront.net/mssql/SQLProMSSQL.#{version}.app.zip

**Comment by adidalal** [comment]
We prefer versioned downloads. Please change them and set the `version` and `shasum` for each.

I commended with the appropriate URLs. Please change accordingly. Keep in mind they will need to be in `"` for string interpolation to correctly work.

Also, they all need a url comment, of the form: (change the url accordingly)
Needs a `# d3fwkemdw8spx3.cloudfront.net/mysql was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by adidalal** [comment]
Also, just noticed you're the developer - if you have an appcast for updates, please add that in as well.

App update submissions to homebrew cask can be semi-automated via `cask-repair (details in [CONTRIBUTING](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md))

**Comment by hankinsoft** [comment]
Sorry, I had not realized non-versioned were the preferred route. I'll fix this shortly. Thank you.

**Comment by hankinsoft** [comment]
All files have been modified per request. At the moment, there are no appcast files available.

**Pull request merged**